be2-scala/src/main/scala/ch/epfl/pop/pubsub/PublishSubscribe.scala

```diff
@@ -52,7 +60,9 @@ object PublishSubscribe {
         val hasChannelPartition = builder.add(ParamsWithChannelHandler.graph(clientActorRef))
 
         val merger = builder.add(Merge[GraphMessage](totalPorts))
+        val broadcast = builder.add(Broadcast[GraphMessage](2))
```

Comment:
For consistency, I think the "2" should be a variable with its corresponding name.